lib/rust/ensogl/component/grid-view/src/entry/visible.rs

```diff
@@ -162,3 +164,109 @@ pub fn set_position<E: display::Object>(
 ) {
     entry.set_position_xy(position(row, col, entry_size, column_widths));
 }
+
+/// Get size of entry at given row and column.
+pub fn size(
+    _row: Row,
+    col: Col,
+    base_entry_size: Vector2,
+    column_widths: &ColumnWidths,
+) -> Vector2 {
+    Vector2(base_entry_size.x + column_widths.width_diff(col), base_entry_size.y)
+}
+
+/// Return the position of the top-left corner of a viewport containing the area around the entry
+/// at given row and column. The area around an entry is defined as the bounding box of the entry
+/// enlarged by given margins. If there is more than one such viewport possible, return the one
+/// closest to the given viewport. The returned viewport has the same size as given viewport.
+///
+/// In the picture below, the dashed border represents the viewport, while the solid border
+/// represents the entry contained in the viewport.
+/// ```text
+/// ┌┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┐
+/// ┆                     ▲             ┆
+/// ┆         margins.top │             ┆
+/// ┆                     ▼             ┆
+/// ┆ margins ┌───────────────┐ margins ┆
+/// ┆  .left  │  entry        │  .right ┆
+/// ┆ ◀─────▶ └───────────────┘ ◀─────▶ ┆
```

Comment:
```suggestion
/// ┆  .left  │  entry        │  .right ┆
/// ┆ ◀───▶ └───────────────┘ ◀───▶ ┆
```

Reply:
As discussed, seems to be a fonts rendering glitch on Adam's side, to be ignored.